server/mscalendar/calendar.go

```diff
@@ -32,7 +29,7 @@ func (m *mscalendar) ViewCalendar(user *User, from, to time.Time) ([]*remote.Eve
 	return m.client.GetDefaultCalendarView(user.Remote.ID, from, to)
 }
 
-func (m *mscalendar) viewTodayCalendar(user *User) ([]*remote.Event, error) {
+func (m *mscalendar) viewTodayCalendar(user *User, timezone string) ([]*remote.Event, error) {
```

Comment:
View sounds like "this will be printed on the screen". Wouldn't it be better "getTodayCalendar" or something similar?

Reply:
Added the change :+1: